terra/compute/container.py

```diff
@@ -177,7 +181,7 @@ def add_file_readonly(self, local, remote, use_local_extension=False):
       remote = os.path.splitext(remote)[0] + local_ext
 
     # update volume
-    self.add_volume_readonly(local, remote, local_must_exist=True)
+    self.add_volume_readonly(local, remote)
```

Comment:
The intent of `add_file_readonly` is to add an _existing_ file as read-only.  Why have you removed the `local_must_exist` check?

Reply:
Because `add_volume_readonly` and `add_file_readonly` (which calls `add_volume_readonly`) do not take `local_must_exist` as an argument. For the reasons you just stated, it shouldn't be an option, but always exist. `add_volume_readonly` does set it, but since `add_file_readonly` is calling `add_volume_readonly` and not `add_volume`, there is no need to repeat `local_must_exist`.